### PR TITLE
First wave of fixes

### PR DESCRIPTION
**Pull request by szepeviktor**
Discovered by @phpstan

### CODE REVIEW

**Comment by szepeviktor** [comment]
@aristath Remaining 2 problems on Level 0.

```
$ packages/bin/phpstan analyze -l 0
Note: Using configuration file /home/viktor/src/gridd/phpstan.neon.dist.
 117/117 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 ------ ------------------------------------------------------
  Line   inc/classes/CSS.php
 ------ ------------------------------------------------------
  159    Access to an undefined property Gridd\CSS::$context.
 ------ ------------------------------------------------------

 ------ --------------------------------
  Line   template-parts/the-content.php
 ------ --------------------------------
  22     Function gridd not found.
 ------ --------------------------------


 [ERROR] Found 2 errors

```

File: grid-parts/Navigation/customizer.php [hunk 1]
```diff
@@ -285,13 +271,6 @@ function gridd_nav_customizer_options( $id ) {
 				'bottom-center' => esc_html__( 'Bottom Center', 'gridd' ),
 				'bottom-right'  => esc_html__( 'Bottom Right', 'gridd' ),
 			],
-			'active_callback'   => [
-				[
-					'setting'  => "nav_{$id}_responsive_behavior",
-					'value'    => 'desktop-normal mobile-hidden',
-					'operator' => '!==',
-				],
-			],
```
Comment (szepeviktor) [inline grid-parts/Navigation/customizer.php:294]: These PHP array duplicate keys have been fixed without knowing them!!

**Comment by szepeviktor** [comment]
and some ideas https://github.com/szepeviktor/gridd/commit/a3916168b2e380e868feb48f5231c752c83cb3c5

**Comment by aristath** [comment]
Perfect, thank you!
Reviewed all changes and merged :heart:

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**